src/module/addRemove.js

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-loop-func */
+let todoList = [];
+let isEditing = false;
+let todoEdit = null;
+
+const saveData = () => {
+  localStorage.setItem('todoslist', JSON.stringify(todoList));
+};
+
+const getStorageData = () => {
+  const localFormData = JSON.parse(localStorage.getItem('todoslist'));
+  if (localFormData === null) {
+    todoList = [];
+  } else {
+    todoList = localFormData;
+  }
+};
+
+const editTodoList = (todo) => {
+  isEditing = true;
+  todoEdit = todo;
+  const desc = document.querySelector('.inputTask');
+  desc.value = todo.description;
+  desc.focus();
+};
+
+const removeTodoList = (indexID) => {
+  todoList = todoList.filter((ind) => ind.index !== indexID);
+  todoList = todoList.map(
+    (todo, index) => (
+      { completed: todo.completed, description: todo.description, index: index + 1 }
+    ),
+  );
+  displayToDo();
+};
+
+const toggleToDoStatus = (todo) => {
+  todoList = todoList.map((todoItem) => {
+    if (todoItem.index === todo.index) {
+      return { ...todo, completed: !todo.completed };
+    }
+    return todoItem;
+  });
+  saveData();
+};
+
+const clearCheckBox = () => {
+  const completedTodoList = todoList.filter((todo) => todo.completed);
+  completedTodoList.forEach((todoItem) => {
+    todoItem.completed = false;
+  });
+  saveData();
+};
+
+const clearCompletedList = () => {
+  todoList = todoList.filter((todo) => !todo.completed);
+  todoList = todoList.map(
+    (todo, index) => (
+      { completed: todo.completed, description: todo.description, index: index + 1 }
+    ),
+  );
+  saveData();
+  displayToDo();
+};
+
+const displayToDo = () => {
+  const todoListElement = document.querySelector('.todo-container');
+  todoListElement.innerHTML = '';
+
+  for (let i = 0; i < todoList.length; i += 1) {
```

Comment:
- [ ] Please consider Using forEach instead of a for loop in displayToDo for iterating over the todoList.
      For simplicity and readability. :tada: :tada:

Reply:
I appreciate your time and effort.
Thank you very much @Ol-create :+1: